metadata.py

```diff
@@ -25,10 +26,10 @@ def appmetadata() -> AppMetadata:
     # first set up some basic information
     metadata = AppMetadata(
         name="Datimex Extraction",
-        description="",  # briefly describe what the purpose and features of the app
-        app_license="",  # short name for a software license like MIT, Apache2, GPL, etc.
+        description="Extracts date expressions from text documents and normalizes them.",
+        app_license="MIT",  # short name for a software license like MIT, Apache2, GPL, etc.
         identifier="datimex-extraction",  # should be a single string without whitespaces. If you don't intent to publish this app to the CLAMS app-directory, please use a full IRI format.
-        url="https://fakegithub.com/some/repository",  # a website where the source code and full documentation of the app is hosted
+        url="https://github.com/clamsproject/app-datimex-extraction",
         # (if you are on the CLAMS team, this MUST be "https://github.com/clamsproject/app-datimex-extraction"
         # (see ``.github/README.md`` file in this directory for the reason)
         analyzer_version='version_X', # use this IF THIS APP IS A WRAPPER of an existing computational analysis algorithm
```

Comment:
this is an optional field, so this line can be remove if there's no off-the-shelf underlying analyzer.